chain-storage-sqlite-old/Cargo.toml

```diff
@@ -12,14 +12,13 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 chain-core = { path = "../chain-core" }
```

Comment:
+ 10 if you can remove that dependency too

Reply:
I don't think it's really possible right now without introducing more breaking changes. With little modification I can stop using `Block` and `BlockId`, but the current implementation relies at least on `Serialize`/`Deserialize` traits from `chain-core`. I think it would be better to work on removing this dependency later in a separate task.

Reply:
https://github.com/input-output-hk/chain-libs/pull/278

Reply:
I suggest to integrate it in a separate task. Replacing `Block` and `BlockId` traits involves a lot of additional refactoring on the `jormungandr` side.